cmds/core/yes/yes_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.Run(m, main)
+}
+
+func TestYes(t *testing.T) {
+	cmd := testutil.Command(t)
+	fmt.Println(cmd.Path)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if err := cmd.Start(); err != nil {
+		log.Fatal(err)
+	}
+
+	in := bufio.NewScanner(stdout)
+
+	for i := 0; i < 20; i++ {
+		if in.Scan() {
+			log.Println(in.Text())
+		} else {
+			log.Println("No output")
+		}
+	}
+	cmd.Process.Kill()
```

Comment:
if err := cmd.Process.Kill(); err != nil {
t.Errorf("Killing child process: got %v, want nil", err)
}

Reply:
Oh, this makes sense. This is a very good point. Thank you!